### ISSUE TITLE
Paid access to >50 requests?

### ISSUE THREAD

**Issue by dick-jo** [openai/dalle-2-preview#18]
Recently received an invite and have begun utilising for an experimental creative project, however the 50 calls per day is prohibitive.

I see that openai provides tiered paid access to several models, but I'm unsure if anything like this is available for Dall-E 2. Is anyone aware of any professional (paid or otherwise) access to uncapped requests?

**Comment by ezzcodeezzlife** [comment]
openai has not said anything about pricing or an api like this :( i guess it will be like gpt3 api pricing where every call costs 0.xx$ and a free trial tier

**Comment by dick-jo** [comment]
This is my best guess so far also. I have a hunch a very select pool of users have unlimited access. Hopefully pricing announcement in the near future as I'm eager to make a significant commitment to the tool for future projects.

**Comment by ezzcodeezzlife** [comment]
if you want to contribute consider checking out [this python package](https://github.com/ezzcodeezzlife/dalle2-in-python) or [this Nextjs webapp](https://github.com/ezzcodeezzlife/DALLE-2-App) or [this npm package](https://github.com/ezzcodeezzlife/dalle-node) 🔥

**Comment by TPS** [comment]
> if you want to contribute consider checking out [this python package](https://github.com/ezzcodeezzlife/dalle2-in-python) or [this Nextjs webapp](https://github.com/ezzcodeezzlife/DALLE-2-App) or [this npm package](https://github.com/ezzcodeezzlife/dalle-node) 🔥

@ezzcodeezzlife To be fair, that's contributing to _your_ dev costs, not this base repo's or @OpenAI's, IIUC.

**Comment by dillon-sykes** [comment]
I have no idea how credible [this tweet](https://twitter.com/bakztfuture/status/1542241519867920386?s=20&t=TTROivkV8bUWW1UCVegqFw) is, but it claims a billing/credit system is currently being built for DALL·E 2.

**Comment by woctezuma** [comment]
> I have no idea how credible [this tweet](https://twitter.com/bakztfuture/status/1542241519867920386?s=20&t=TTROivkV8bUWW1UCVegqFw) is, but it claims a billing/credit system is currently being built for DALL·E 2.

That would not be surprising. That is what OpenAI has done for GPT-3, I believe.

**Comment by avincodes** [comment]
Yep as predicted Open Ai has come up with a pricing model where users can purchase blocks of 115 additional credits (1 credit = 1 prompt) for 15$ (USD).
![image](https://user-images.githubusercontent.com/66766682/181916358-68e37d3c-e947-4d8a-9a27-a7b70f346904.png)

**Comment by woctezuma** [comment]
AI as a Service.

**Comment by kieradev** [comment]
> AI as a Service.

Using images from the internet without paying to train an AI to generate images which are then sold to the internet.
